mco_ovs_workers.yml

```diff
@@ -25,7 +25,7 @@ spec:
+            source: data:text/plain;charset=utf-8;base64,${base64_script_content}
```

Comment:
I tried generating the content and the result is not valid. It renders it as multiline without needed yaml indentation. I suggest you change the base64 command so it does not use newlines.

Reply:
Try again and update, commend changed

Reply:
I checked out the current form, copy pasted the command and it ended up removing the ${...} but has not replace it by any base64.

Reply:
@phoracek Now after all the changes, please check to make sure the command is working like it should